language/stdlib/modules/Libra.move

```diff
@@ -630,6 +644,14 @@ module Libra {
         assert_assoc_and_currency<FromCoinType>(account);
         let currency_info = borrow_global_mut<CurrencyInfo<FromCoinType>>(CoreAddresses::CURRENCY_INFO_ADDRESS());
         currency_info.to_lbr_exchange_rate = lbr_exchange_rate;
+        Event::emit_event(
+            &mut currency_info.exchange_rate_update_events,
+            ToLBRExchangeRateUpdateEvent {
+                currency_code: *&currency_info.currency_code,
```

Comment:
@tnowacki: does it work to write `copy currency_info.currency_code` here (and if it does, is there a reason to prefer one of `copy` and `*&`?

Reply:
I actually wound up writing `copy currency_info.currency_code` here at first, but the parser was unhappy with that. I didn't try `copy &currency_info.currency_code`, but I'm guessing the compiler wants to see ` &` in front of a field access to know that it's grabbing a reference to that field.